krux/cli.py

```diff
@@ -110,11 +114,8 @@ def __init__(self, name, parser=None, logger=None, lockfile=False):
         ### and parse them
         self.args = self.parser.parse_args()
 
-        ### get a logger - is there a way to to have logger be relative to the
-        ### invocation of the log call?
-        self.logger = logger or krux.logging.get_logger(
-            name, level=self.args.log_level
-        )
+        self.logger = logger
```

Comment:
Or just don't set the logger directly in init, pass the logger through, and let _init_logger() do it.